src/maison/config_sources/toml_source.py

```diff
@@ -1,10 +1,13 @@
 """Module to hold the `TomlSource` class definition."""
 
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib
```

Comment:
I think you'd need to add `tomli` as a dependency to the project which you can do with `uv add toml`. If you add that then you can remove this `try ... except` which would then mean that you can undo the `fail_under` change back to 100. If this package eventually removes support for python 3.9 and 3.10 then we can remove the `tomli` dependency and use `tomlib`.

Reply:
Or we can do this suggestion [here](https://github.com/hukkin/tomli?tab=readme-ov-file#building-a-tomlitomllib-compatibility-layer) and add `tomli` as a dependency if the python version is less than 3.11 which you can do with
````
uv add "tomli; python_version < '3.11'"
```

Reply:
here it is. nice guide